script/draft-release/run.ts

```diff
@@ -115,6 +129,10 @@ export async function run(args: ReadonlyArray<string>): Promise<void> {
   })
   const nextVersion = getNextVersionNumber(previousVersion, channel)
 
+  console.log(`Creating release branch for "${nextVersion}"...`)
+  createReleaseBranch(nextVersion)
+  console.log(`Done!`)
```

Comment:
I think this is the most controversial change from this PR. What do you think about it?

Reply:
I am all for scripting this repetitive step!

Reply:
I love it :shipit: